packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -894,10 +964,20 @@ export default function<T, P, I, TI, HI, PI, C, CC, CX, PL>(
       } else {
         // The root did not complete.
         invariant(
-          false,
+          !nextRenderIsExpired,
           'Expired work should have completed. This error is likely caused ' +
```

Comment:
FYI, I got here with `16.4.0-alpha.3174632`

Reply:
Yeah that’s the bug @ryanflorence found. Should be fixed now, haven’t pushed a new canary yet.